.github/workflows/issue_pr_lock.yml

```diff
@@ -48,9 +50,20 @@ jobs:
       issues: write
       pull-requests: write
     steps:
+      # Use dedicated github app to workaround API rate limiting
+      # Ref: https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/making-authenticated-api-requests-with-a-github-app-in-a-github-actions-workflow
+      - name: Obtain Stale Locking App token
+        id: generate-token
+        uses: actions/create-github-app-token@v1
+        with:
+          # N/B: These are both defined at the containers-org level
+          app-id: ${{ vars.STALE_LOCKING_APP_ID }}
```

Comment:
The string `vars` does not appear anywhere else in `.github/workflows`; the usual form is `secrets.XX`. Is `vars` a real thing, or did you mean `secrets` here?

Reply:
Sharp eye.  This was surprising to me also (new feature maybe?).  Yes it's intended and [documented that it will consider org-level action-vars](https://docs.github.com/en/actions/learn-github-actions/contexts#vars-context) (which this is).